perma_web/frontend/components/ActionButton.vue

```diff
@@ -0,0 +1,34 @@
+<script setup>
+
+const props = defineProps({
+  name: String,
+  handleClick: Function,
+  iconFamily: {
+    type: String,
+    default: "fontAwesome",
+    validator(value) {
+      return ["fontAwesome", "none"].includes(value);
+    },
+  },
+  icon: {
+    type: String,
+    validator(value) {
+      return ["download-alt", "plus", "edit", "trash", "chevron-down", "chevron-up"].includes(value);
+    },
```

Comment:
I didn't include "search" as one of the icon options here, because it differs in more than a couple of ways in use from these other icons — it's a smaller size, etc. 

It should be easy to add it in if we decide to reuse the `ActionButton` component within the search functionality on the page.